src/common/components/ui/OpenShiftVersionDropown.tsx

```diff
@@ -79,20 +84,30 @@ export const OpenShiftVersionDropdown = ({
     ),
     [setOpen, current, isDisabled],
   );
-
+  const location = useLocation();
+  console.log(location);
```

Comment:
Looks like `useLocation` is not used since ultimately you're using `window.location`, no?

Reply:
is a mistake. I've deleted it. I'm using window.location